Enterprise-CMCS/eAPD
# dmirano/4509-outcomes-and-metrics-help-text

Resolves #4509 

### Description
Changes the help text for the outcomes and metrics page for mmis only.

### Significant changes or possible side effects
Only affects outcomes.js and the corresponding yaml file

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |

### Steps to manually verify this change

1. Verify HITECH has old help text
2. Verify MMIS has the new help text

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
